variants.csv

```diff
@@ -0,0 +1,10 @@
+WBPaper ID,Oligo,Sentence
```

Comment:
Please remove example outputs like this csv file

Reply:
These files are been currently used in text extraction and settings file, so cannot remove them.

Reply:
They can be removed if you remove the settings.py file and pass the token only (perhaps loaded from an environmental variable?